grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx

```diff
@@ -346,6 +356,16 @@ export const IntegrationForm = observer(
                 <div className={styles.webhookSwitch}>
                   <Switch value={field.value} onChange={field.onChange} />
                   <Text type="primary"> Create default outgoing webhooks</Text>
+                  <Tooltip
+                    content={
+                      <>
+                        If enabled, all the necessary webhooks will be created automatically. It's highly recommended to
+                        keep this option enabled.
+                      </>
+                    }
+                  >
+                    {<Icon name={'info-circle'} />}
```

Comment:
I think curly brackets are not needed here? They will be rendered on UI

Reply:
https://github.com/grafana/oncall/pull/4229/commits/cbcf15773d66ac5929d68137bd55798a1b8c33e3